src/fu-engine.c

```diff
@@ -6354,7 +6354,7 @@ fu_engine_load (FuEngine *self, FuEngineLoadFlags flags, GError **error)
 		return TRUE;
 
 /* TODO: Read registry key [HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Cryptography] "MachineGuid" */
-#ifndef _WIN32
+#ifdef __linux__
```

Comment:
is there anything else similar that can be obtained from `*BSD`?

Reply:
@superm1 I found out how we can recognize if dbus is enabled. The `machine-id` exists only if dbus is running, so we can check if the file is present during meson configuration. In addition, I will add the one more `fns` path (`/var/db/dbus/machine-id`) to `fwupd_build_machine_id`.

Reply:
Sounds good.